### DISCUSSION TITLE
Qexo目录和文章扫描深度不够

### DISCUSSION THREAD

**Discussion by Fgaoxing** [Qexo/Qexo#263] [General]
再打开文章和页面列表时，扫描深度不够，倒是页面的子页面没法编辑
解决办法：递归查找或限定格式查找，不过建议递归查找，好处是即使不是.md和.markdown也能看到

**Comment by Fgaoxing** [comment]
@am-abudu

**Comment by am-abudu** [comment]
页面还能有子页面吗，我没有考虑这点
是否能给个示例

**Reply by Fgaoxing** [reply]
@am-abudu https://www.yt-blog.top/docs/

**Reply by am-abudu** [reply]
这是一个404页面，我不太明白

**Reply by Fgaoxing** [reply]
@am-abudu 你再试试
